src/main/java/de/refactoringbot/refactoring/supportedrefactorings/RenameMethod.java

```diff
@@ -129,12 +129,13 @@ public String performRefactoring(BotIssue issue, GitConfiguration gitConfig) thr
 
 		// If refactor-method not found
 		if (methodToRefactor == null) {
-			throw new BotRefactoringException("Could not find specified method! Automated refactoring failed.");
+			throw new BotRefactoringException(
+					"Could not find specified method declaration at given line! Please check if you placed your comment on a method declaration.");
 		}
 
 		// If new name equals old
 		if (oldMethodName != null && oldMethodName.equals(issue.getRefactorString())) {
-			throw new BotRefactoringException("New method name must differ from the current one!");
+			throw new BotRefactoringException("New method name must differ from the current one! Please create a new comment with a different method name.");
```

Comment:
Same issue with the second part of the message as already mentioned in a previous comment.

Reply:
What if we remove that part and the user just edits his comment instead of creating a new one? The bot won't look at the same comment again. I don't know how to make it more general without making the next step less clear.

We could tell the refactoring class that we have a comment refactoring, and if we do, we add a more specific comment to the exception.